react-design/my-app/src/Indicators.tsx

```diff
@@ -0,0 +1,31 @@
+type Props = {
+  items: string[];
+  highlighted: number;
+  handleClick: (e: number) => void;
+};
+
+export function Indicators({ items, highlighted, handleClick }: Props) {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        gap: '1rem',
+      }}>
+      {items.map((_, index) => (
+        <div
```

Comment:
Why not use `<button>`? For accessibility reasons, it's best to use the element that represents the purpose.